tools/c_ast/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "c_ast"
+version = "0.1.0"
+edition = "2024"
+
+[dependencies]
+clang-ast = { version = "0.1.35", features = ["serde"] }
+full_source = { version = "0.1.0", path = "../full_source" }
+harvest_core.workspace = true
+serde = { workspace = true, features = ["derive"] }
```

Comment:
The workspace's `serde` already includes `features = ["derive"]`, so that does not need to be repeated here.